content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
+all the benefits of this are not visible for you in the daily use just yet, it's a necessary step towards a better application. In fact, it already paid off in application speed. See this video:
+
+<video autoplay muted loop preload="none" id="preview">
+    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
+</video>
+
+Not only you're seeing the speed differences, you're seeing the new animations! Images **will fade in** every time you move to another page. You **can** disable that in settings if you want to come back the old behaviour though.
+
+The old-standing issue that many of you had (by the way, introduced by the *fellow writer* of this post 🙄) caused by the images has been fixed in this version
+(although we want to keep improving in that matter, as we're still not completely happy with the current implementation) 
+
+That's pretty much what you'll notice at first sight for web anyway. A lot of not-that-relevant improvements, translations and bug fixes have been added. Consider that they're easter eggs, so happy finding! Ah, and we also added keyboard support for TVs and improved the compatibility with old browsers (just in case)!
+
+Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).
+Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):
+
+```
+|                         Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
+|------------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
+|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
+|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
+```
+*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*
+
+That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.
+However, there are some key improvements here, which are mostly focused in performance:
+
+* Added support for embedded attachments in media files (#1838)
+* Fixed Play To order for playlists
+* Raspberry Pi should have now full hardware-acceleration capabilities when encoding and decoding (older models, not in 4 yet)
+* General improvements to the transcoding speed and reliability
+* Reliability improvements
+* Tons of bug fixes
+
+## Extra, Extra!
+
+Along the way, we also updated a lot of our documentation. Specifically, the experts have been working hard in revamping the **network/reverse proxy**
+and the **hardware acceleration** configuration guides. Not only improvements have been made, but also new sections have been added to the documentation, such
+as the extensive **codec** and the **custom CSS branding** guides.
```

Comment:
We should link directly to these specific pages in the documentation, where possible